constants.xml

```diff
@@ -146,6 +146,7 @@
         <STATUS_RPC_PORT>4301</STATUS_RPC_PORT>
         <IP_TO_BIND>127.0.0.1</IP_TO_BIND>
         <ENABLE_STATUS_RPC>true</ENABLE_STATUS_RPC>
+        <SCILLA_IPC_SOCKET_PATH>/tmp/scillaipcsocket</SCILLA_IPC_SOCKET_PATH>
```

Comment:
how about `/tmp/zilliqa.sock`?

People know Unix domain socket is normally being used for IPC so we can save the word `ipc`.

Not suggesting calling `scilla.sock` because it's actually a listening socket opened by zilliqa process